Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -342,6 +342,7 @@ DECLARE_SOA_COLUMN(Chi2MatchMCHMFT, chi2MatchMCHMFT, float);   //! MCH-MFT Match
 DECLARE_SOA_COLUMN(MatchScoreMCHMFT, matchScoreMCHMFT, float); //! MCH-MFT Machine Learning Matching Score for GlobalMuonTracks
 DECLARE_SOA_COLUMN(MatchMFTTrackID, matchMFTTrackID, int);     //! ID of matching MFT track for GlobalMuonTrack (ints while self indexing not available)
 DECLARE_SOA_COLUMN(MatchMCHTrackID, matchMCHTrackID, int);     //! ID of matching MCH track for GlobalMuonTracks  (ints while self indexing not available)
+DECLARE_SOA_COLUMN(MCHBitMap, MchBitMap, short);               //! Fired muon chambers BitMap
```

Comment:
Since you use it as a bit map, you should make this as unsigned, in this case uint16_t